### PR TITLE
Events Page Update + Add Upcoming Events

### LINKED ISSUE #244: Add events to the website
Hi there, could you please add the two following events to the website :) 

https://www.meetup.com/women-coding-community/events/301218272/

https://www.meetup.com/women-coding-community/events/301301532/

Many thanks!

### PR DESCRIPTION
**Pull request by khairahscorner**
## Description
- Added two upcoming events
- Updated logic for events page to show past and upcoming events in appropriate tabs
- Updated home page to feature latest two upcoming events and remove section if no upcoming events

## Change Type
- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [x] Other

## Related Issue
Resolves #244 

## Screenshots
After:

Screenshot 2024-06-02 at 11 12 47 am
Screenshot 2024-06-02 at 11 48 22 am
Screenshot 2024-06-02 at 11 48 28 am

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -32,3 +32,38 @@
...
+    path: https://www.meetup.com/women-coding-community/events/301218272/
+    title: View meetup event
+    target: _blank
+- title: |
+    Building Confidence in the Workplace
+  description: | 
+    Join Women Coding Community for the great discussion which will provide practical strategies and tips to help you develop and enhance your confidence in the workplace. The aim of the talk is to empower you to feel confident in your abilities, improve your performance and achieve success in your professional and business life!
+  category: category-career-talk
+  category_name: Career Talk
+  date: TUES, JUL 23, 2024
```
Comment (lilyndk) [inline _data/events.yml]:
```suggestion
  date: TUE, JUL 23, 2024
```

File: _includes/featured-events.html [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-            <h2>Events</h2>
-        </div>
-        <div class="row-content col-12">
-            <div class="card-deck">
-                {% assign count = 0 %}
-                {% for event in site.data.events %}
-                    {% assign count = count | plus: 1 %}
-                    {% if count <= 2 %}
+{% assign today = 'now' | date: "%Y%m%d" %}
+{% assign valid_events = site.data.events | where_exp: "event", "event.expiration >= today" %}
```
Comment (dricazenck) [inline _includes/featured-events.html:2]:
```suggestion
{% assign valid_events = site.data.events | where_exp: "event", "event.expiration > today" %}
```

File: events.html [hunk 3]
```diff
@@ -22,17 +24,30 @@
...
+                    <a class="nav-link" id="tab-past-events">Past Events</a>
                 </li>
             </ul>
         </div>
-        <div class="row-content col-12">
+
+        <div class="row-content col-12" id="upcoming-events">
+            <div class="card-deck">
+                {% for event in site.data.events %}
+                    {% if event.expiration >= today %}
```
Comment (dricazenck) [inline events.html:38]:
```suggestion
                    {% if event.expiration > today %}
```

**Review by dricazenck — APPROVED** [review]
🚀  Thank you!!! ❤️

**Pull request merged**